dotnet-interactive/HttpApiBootstrapper.cs

```diff
@@ -10,36 +10,38 @@ internal static class HttpApiBootstrapper
     {
         public static string GetJSCode(Uri apiRoot)
         {
-            var id = $"{Process.GetCurrentProcess().Id}.{apiRoot.Port}";
             var template = @"#!javascript
-if ((typeof(requirejs) !==  typeof(Function)) || (typeof(requirejs.config) !== typeof(Function))) { 
-    let script = document.createElement(""script""); 
-    script.setAttribute(""src"", ""https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js""); 
+if ((typeof(require) !==  typeof(Function)) || (typeof(require.config) !== typeof(Function))) {
+    let script = document.createElement('script');
+    script.setAttribute('src', 'https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js');
     script.onload = function(){
         loadDotnetInteractiveApi();
     };
-    document.getElementsByTagName(""head"")[0].appendChild(script); 
+    document.getElementsByTagName('head')[0].appendChild(script);
 }
 else {
     loadDotnetInteractiveApi();
 }
 
 function loadDotnetInteractiveApi(){
-    let apiRequire = requirejs.config({context:""dotnet-interactive.$SEED$"",paths:{dotnetInteractive:""$API_URL$""}});
```

Comment:
double check since juoyter lab has require but not require js and require.js ints require (if not dfined) and requirejs. in the original code requirejs was the safe api to invoke.

Reply:
I can go back to `requirejs`.  I was running into issues with `require(['dotnet-interactive/foo'], ...)` resolving with the `context` and `$SEED$` values, but if it's a cache buster then I'll go back to it and try to get it working correctly.

Reply:
I had issues with xplot injecting the right require api in jupyter lab.